### PR TITLE
update recipe

### PR DESCRIPTION
**Pull request by ocefpaf**
@conda-forge-admin, please rerender

### CODE REVIEW

**Comment by ocefpaf** [comment]
@bgruening this PR aims to sync our recipe with Anaconda's version.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,14 +36,20 @@ test:
...
 about:
   home: https://cnswww.cns.cwru.edu/php/chet/readline/rltop.html
-  license: GPL3
+  license: GPL-3.0
+  license_file: COPYING
   summary: library for editing command lines as they are typed in
+  description: |
+    The standard Python readline extension statically linked against the GNU
+    readline library.
+  doc_url: https://pypi.python.org/pypi/readline
```
Comment (bgruening) [inline recipe/meta.yaml]: I'm confused by this. Why do we reference python-gnureadline?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -6,19 +6,23 @@ package:
 
 source:
   url: ftp://ftp.gnu.org/gnu/readline/readline-{{ version }}.tar.gz
-  fn: readline-{{ version }}.tar.gz
   sha256: 750d437185286f40a369e1e4f4764eda932b9459b5ec9a731628393dd3d32334
   patches:
     - patches/readline70-001
+    - patches/readline70-002
```
Comment (bgruening) [inline recipe/meta.yaml:12]: There are now a few more patches available at: ftp://ftp.gnu.org/gnu/readline/readline-7.0-patches/

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml]: I copied-n-pasted without checking. It is wrong upstream! Thanks!

File: README.md [hunk 3]
```diff
@@ -3,12 +3,14 @@ About readline
...
 Home: https://cnswww.cns.cwru.edu/php/chet/readline/rltop.html
 
-Package license: GPL3
+Package license: GPL-3.0
 
 Feedstock license: BSD 3-Clause
 
 Summary: library for editing command lines as they are typed in
 
+The standard Python readline extension statically linked against the GNU
```
Comment (bgruening) [inline README.md]: Here as well. I wonder if this was auto generated upstream or manually and if manually why?

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: README.md (hunk 3 above)
Comment (ocefpaf) [reply README.md]: Auto-generated and the re-render below should take care of it.

**Review by bgruening — APPROVED** [review]

**Pull request merged**